scripts/generate_colors.lua

```diff
@@ -104,23 +115,21 @@ end
 --------------------------------------------------------------------------------
 -- Update cterm_color for dark background
 --------------------------------------------------------------------------------
-vim.cmd "noswapfile drop lua/nvim-web-devicons/icons-default.lua"
-
-io.write "Generating cterm colors for dark background..."
-
-iterate_colors(generate_cterm)
-
-vim.cmd "silent! wall!"
-io.write " OK\n"
+for _, file in ipairs(_G.ICON_FILES) do
+  vim.cmd(string.format("noswapfile drop lua/nvim-web-devicons/default/%s", file))
+  io.write(string.format("Generating cterm colors for dark background: %s...", file))
+  iterate_colors(generate_cterm)
+  vim.cmd "silent! wall!"
+  io.write " OK\n"
+end
 
 --------------------------------------------------------------------------------
 -- Update color and cterm_color for light backgrounds
 --------------------------------------------------------------------------------
-vim.cmd "noswapfile drop lua/nvim-web-devicons/icons-light.lua"
-
-io.write "Generating colors for light background..."
-
-iterate_colors(generate_for_light_bg)
-
-vim.cmd "silent! wall!"
-io.write " OK\n"
+for _, file in ipairs(_G.ICON_FILES) do
+  vim.cmd("noswapfile drop lua/nvim-web-devicons/light/" .. file)
+  io.write("Generating colors for light background: " .. file .. "...")
+  iterate_colors(generate_for_light_bg)
```

Comment:
```suggestion
  iterate_colors(generate_for_light_bg)
    vim.cmd(
    string.format(
      "1s/.*/&      -- this file is generated from lua\\/nvim-web-devicons\\/default\\/%s, please do not edit",
      file
    )
  )
```

Reply:
Neat. Will add now. Do we also want to let GH know about it?
https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github